py/desispec/scripts/qa_prod.py

```diff
@@ -14,8 +14,12 @@ def parse(options=None):
                         help = 'Path containing the exposures/directory to use')
     parser.add_argument('--remake_frame', type = int, default = 0,
                         help = 'Bitwise flag to control remaking the QA files (1) and figures (2) for each frame in the production')
-    #parser.add_argument('--outfile', type = str, default = None, required=True,
-    #                    help = 'path of DESI sky fits file')
+    parser.add_argument('-slurp', default = False, action='store_true',
```

Comment:
Ok.  I was trying to distinguish between switches and inputs, but 
there is no real need. 
Updated